tasks/prime.py

```diff
@@ -347,3 +347,27 @@ def create_move_task_order(self):
 
         if success:
             self.set_prime_data(PrimeObjects.MOVE_TASK_ORDER, mto_data)
+
+    @tag(PrimeObjects.MTO_SERVICE_ITEM.value, "updateMTOServiceItemStatus")
+    @task
+    def update_mto_service_item_status(self):
+        mto_service_item = self.get_random_data(PrimeObjects.MTO_SERVICE_ITEM)
+        # if we don't have an mto shipment we can't run this task
+        if not mto_service_item:
+            return
+
+        payload = self.fake_request("/service-items/{mtoServiceItemID}/status", "patch")
+        headers = {"content-type": "application/json", "If-Match": mto_service_item["eTag"]}
+
+        resp = self.client.patch(
+            support_path(f"/service-items/{mto_service_item['id']}/status"),
+            name=support_path("/service-items/:mtoShipmentID/status"),
```

Comment:
Looks like the ID part here still needs to be changed to `{mtoServiceItemID}` instead of `:mtoShipmentID`

Reply:
Thanks for this! I've updated it!